### PR TITLE
MAIN-10371: Update PHPUnit to 6.1

### PR DESCRIPTION
**Pull request by mszabo-wikia**
https://wikia-inc.atlassian.net/browse/MAIN-10371

### CODE REVIEW

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 1]
```diff
@@ -932,10 +769,7 @@ function assertLessThan($expected, $actual, $message = '')
  */
 function assertLessThanOrEqual($expected, $actual, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:770]: MAJOR Remove the unused function parameter "$expected". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 2]
```diff
@@ -309,10 +263,7 @@ function assertAttributeLessThan($expected, $actualAttributeName, $actualClassOr
  */
 function assertAttributeLessThanOrEqual($expected, $actualAttributeName, $actualClassOrObject, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:264]: MAJOR Remove the unused function parameter "$expected". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actualAttributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actualClassOrObject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 3]
```diff
@@ -623,10 +520,7 @@ function assertEmpty($actual, $message = '')
  */
 function assertEqualXMLStructure(DOMElement $expectedElement, DOMElement $actualElement, $checkAttributes = false, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:521]: MAJOR Remove the unused function parameter "$expectedElement". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actualElement". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkAttributes". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 4]
```diff
@@ -1258,10 +1032,7 @@ function assertSame($expected, $actual, $message = '')
  */
 function assertSameSize($expected, $actual, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1033]: MAJOR Remove the unused function parameter "$expected". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 5]
```diff
@@ -1580,10 +1291,7 @@ function atLeastOnce()
  */
 function attribute(Constraint $constraint, $attributeName)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1292]: MAJOR Remove the unused function parameter "$constraint". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$attributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 6]
```diff
@@ -946,10 +780,7 @@ function assertLessThanOrEqual($expected, $actual, $message = '')
  */
 function assertFinite($actual, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:781]: MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 7]
```diff
@@ -1888,10 +1551,7 @@ function isTrue()
  */
 function isType($type)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1552]: MAJOR Remove the unused function parameter "$type". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 8]
```diff
@@ -1273,10 +1044,7 @@ function assertSameSize($expected, $actual, $message = '')
  */
 function assertStringEndsNotWith($suffix, $string, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1045]: MAJOR Remove the unused function parameter "$suffix". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$string". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 9]
```diff
@@ -960,10 +791,7 @@ function assertFinite($actual, $message = '')
  */
 function assertInfinite($actual, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:792]: MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 10]
```diff
@@ -329,10 +280,7 @@ function assertAttributeLessThanOrEqual($expected, $actualAttributeName, $actual
  */
 function assertAttributeNotContains($needle, $haystackAttributeName, $haystackClassOrObject, $message = '', $ignoreCase = false, $checkForObjectIdentity = true, $checkForNonObjectIdentity = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:281]: MAJOR Remove the unused function parameter "$needle". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackAttributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackClassOrObject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForNonObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 11]
```diff
@@ -642,10 +536,7 @@ function assertEqualXMLStructure(DOMElement $expectedElement, DOMElement $actual
  */
 function assertEquals($expected, $actual, $message = '', $delta = 0.0, $maxDepth = 10, $canonicalize = false, $ignoreCase = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:537]: MAJOR Remove the unused function parameter "$expected". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$delta". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$maxDepth". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$canonicalize". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 12]
```diff
@@ -1903,10 +1563,7 @@ function isType($type)
  */
 function lessThan($value)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1564]: MAJOR Remove the unused function parameter "$value". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 13]
```diff
@@ -1602,10 +1310,7 @@ function attribute(Constraint $constraint, $attributeName)
  */
 function attributeEqualTo($attributeName, $value, $delta = 0.0, $maxDepth = 10, $canonicalize = false, $ignoreCase = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1311]: MAJOR Remove the unused function parameter "$attributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$value". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$delta". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$maxDepth". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$canonicalize". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 14]
```diff
@@ -1288,10 +1056,7 @@ function assertStringEndsNotWith($suffix, $string, $message = '')
  */
 function assertStringEndsWith($suffix, $string, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1057]: MAJOR Remove the unused function parameter "$suffix". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$string". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 15]
```diff
@@ -974,10 +802,7 @@ function assertInfinite($actual, $message = '')
  */
 function assertNan($actual, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:803]: MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 16]
```diff
@@ -658,10 +549,7 @@ function assertEquals($expected, $actual, $message = '', $delta = 0.0, $maxDepth
  */
 function assertNotTrue($condition, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:550]: MAJOR Remove the unused function parameter "$condition". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 17]
```diff
@@ -348,10 +296,7 @@ function assertAttributeNotContains($needle, $haystackAttributeName, $haystackCl
  */
 function assertAttributeNotContainsOnly($type, $haystackAttributeName, $haystackClassOrObject, $isNativeType = null, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:297]: MAJOR Remove the unused function parameter "$type". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackAttributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackClassOrObject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$isNativeType". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 18]
```diff
@@ -1920,10 +1577,7 @@ function lessThan($value)
  */
 function lessThanOrEqual($value)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1578]: MAJOR Remove the unused function parameter "$value". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 19]
```diff
@@ -1617,10 +1322,7 @@ function attributeEqualTo($attributeName, $value, $delta = 0.0, $maxDepth = 10,
  */
 function callback($callback)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1323]: MAJOR Remove the unused function parameter "$callback". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 20]
```diff
@@ -1306,10 +1071,7 @@ function assertStringEndsWith($suffix, $string, $message = '')
  */
 function assertStringEqualsFile($expectedFile, $actualString, $message = '', $canonicalize = false, $ignoreCase = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1072]: MAJOR Remove the unused function parameter "$expectedFile". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actualString". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$canonicalize". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 21]
```diff
@@ -992,10 +817,7 @@ function assertNan($actual, $message = '')
  */
 function assertNotContains($needle, $haystack, $message = '', $ignoreCase = false, $checkForObjectIdentity = true, $checkForNonObjectIdentity = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:818]: MAJOR Remove the unused function parameter "$needle". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystack". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForNonObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 22]
```diff
@@ -674,10 +562,7 @@ function assertNotTrue($condition, $message = '')
  */
 function assertFalse($condition, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:563]: MAJOR Remove the unused function parameter "$condition". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 23]
```diff
@@ -365,10 +310,7 @@ function assertAttributeNotContainsOnly($type, $haystackAttributeName, $haystack
  */
 function assertAttributeNotCount($expectedCount, $haystackAttributeName, $haystackClassOrObject, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:311]: MAJOR Remove the unused function parameter "$expectedCount". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackAttributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackClassOrObject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 24]
```diff
@@ -1632,10 +1334,7 @@ function callback($callback)
  */
 function classHasAttribute($attributeName)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1335]: MAJOR Remove the unused function parameter "$attributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 25]
```diff
@@ -1321,10 +1083,7 @@ function assertStringEqualsFile($expectedFile, $actualString, $message = '', $ca
  */
 function assertStringMatchesFormat($format, $string, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1084]: MAJOR Remove the unused function parameter "$format". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$string". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 26]
```diff
@@ -1008,10 +830,7 @@ function assertNotContains($needle, $haystack, $message = '', $ignoreCase = fals
  */
 function assertNotContainsOnly($type, $haystack, $isNativeType = null, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:831]: MAJOR Remove the unused function parameter "$type". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystack". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$isNativeType". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 27]
```diff
@@ -1948,10 +1599,7 @@ function logicalAnd()
  */
 function logicalNot(Constraint $constraint)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1600]: MAJOR Remove the unused function parameter "$constraint". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 28]
```diff
@@ -692,10 +577,7 @@ function assertFalse($condition, $message = '')
  */
 function assertFileEquals($expected, $actual, $message = '', $canonicalize = false, $ignoreCase = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:578]: MAJOR Remove the unused function parameter "$expected". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$actual". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$canonicalize". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$ignoreCase". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 29]
```diff
@@ -381,10 +323,7 @@ function assertAttributeNotCount($expectedCount, $haystackAttributeName, $haysta
  */
 function assertAttributeNotEmpty($haystackAttributeName, $haystackClassOrObject, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:324]: MAJOR Remove the unused function parameter "$haystackAttributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystackClassOrObject". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 30]
```diff
@@ -1648,10 +1347,7 @@ function classHasAttribute($attributeName)
  */
 function classHasStaticAttribute($attributeName)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1348]: MAJOR Remove the unused function parameter "$attributeName". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 31]
```diff
@@ -1336,10 +1095,7 @@ function assertStringMatchesFormat($format, $string, $message = '')
  */
 function assertStringMatchesFormatFile($formatFile, $string, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1096]: MAJOR Remove the unused function parameter "$formatFile". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$string". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 32]
```diff
@@ -76,10 +72,7 @@ function anything()
  */
 function arrayHasKey($key)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:73]: MAJOR Remove the unused function parameter "$key". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 33]
```diff
@@ -1023,10 +842,7 @@ function assertNotContainsOnly($type, $haystack, $isNativeType = null, $message
  */
 function assertNotCount($expectedCount, $haystack, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:843]: MAJOR Remove the unused function parameter "$expectedCount". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$haystack". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 34]
```diff
@@ -706,10 +588,7 @@ function assertFileEquals($expected, $actual, $message = '', $canonicalize = fal
  */
 function assertFileExists($filename, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:589]: MAJOR Remove the unused function parameter "$filename". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 35]
```diff
@@ -1666,10 +1362,7 @@ function classHasStaticAttribute($attributeName)
  */
 function contains($value, $checkForObjectIdentity = true, $checkForNonObjectIdentity = false)
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:1363]: MAJOR Remove the unused function parameter "$value". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$checkForNonObjectIdentity". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php [hunk 36]
```diff
@@ -91,10 +84,7 @@ function arrayHasKey($key)
  */
 function assertArrayHasKey($key, $array, $message = '')
```
Comment (WikiaTech) [inline lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php:85]: MAJOR Remove the unused function parameter "$key". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$array". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
MAJOR Remove the unused function parameter "$message". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

**Comment by mszabo-wikia** [comment]
Wow, what a mess. I will exclude `lib/composer` from the Sonar job.

**Review by macbre — APPROVED** [review]

**Pull request merged**